### PR TITLE
Update developer guide

### PR DESCRIPTION
**Pull request by hickeyma**
Add some improvements to building a local environment section.

**Which issue(s) this PR fixes**:
Adds improvement to the developer guide to help users.

### CODE REVIEW

File: contributors/devel/development.md [hunk 1]
```diff
@@ -136,8 +136,8 @@ platform.
 
 ### jq
 
-Kube-apiserver requires the `jq` to be installed to successfully build your Local Kubernetes Deployment. 
-You can see installation guide for `jq` over [here](https://stedolan.github.io/jq/download/).
+Kube-apiserver requires `jq` to be installed to successfully build your local Kubernetes environment.
```
Comment (mikebrow) [inline contributors/devel/development.md:139]: s/build your local Kubernetes environment/build Kubernetes in your local environment/

File: contributors/devel/development.md [hunk 2]
```diff
@@ -136,8 +136,8 @@ platform.
 
 ### jq
 
-Kube-apiserver requires the `jq` to be installed to successfully build your Local Kubernetes Deployment. 
-You can see installation guide for `jq` over [here](https://stedolan.github.io/jq/download/).
+Kube-apiserver requires `jq` to be installed to successfully build your local Kubernetes environment. 
+You can see the installation guide for `jq` [here](https://stedolan.github.io/jq/download/).
```
Comment (mikebrow) [inline contributors/devel/development.md:140]: suggest: The [`jq` installation guide]()] provides detailed instructions for supported platforms.

File: contributors/devel/development.md [hunk 3]
```diff
@@ -197,9 +203,10 @@ If everything checks out you will have an executable in the `_output/bin` direct
 
 *Note:* If you are using `CDPATH`, you must either start it with a leading colon, or unset the variable. The make rules and scripts to build require the current directory to come first on the CD search path in order to properly navigate between directories.
 
+To build everything:
 ```sh
 cd $working_dir/kubernetes
-make
+make all
```
Comment (mikebrow) [inline contributors/devel/development.md:209]: make is equivalent to make all...
suggest saying something to the effect of make or make all..?

File: contributors/devel/development.md (hunk 1 above)
Comment (hickeyma) [reply contributors/devel/development.md:139]: done

File: contributors/devel/development.md (hunk 2 above)
Comment (hickeyma) [reply contributors/devel/development.md:140]: Good suggestion, updated.

File: contributors/devel/development.md (hunk 3 above)
Comment (hickeyma) [reply contributors/devel/development.md:209]: Forgot the comment. Thanks for the reminder.

**Comment by hickeyma** [comment]
Thanks for the review @mikebrow. I have added the changes and its ready for review again.

**Comment by hickeyma** [comment]
Thanks @spiffxp

**Pull request merged**